pwa/app/routes/_index.tsx

```diff
@@ -12,11 +14,34 @@ export async function loader() {
     });
   }
 
+  const events = await getValidEvents(status.data.current_season);
+
   return json({
     status: status.data,
+    events: events,
   });
 }
 
+export async function getValidEvents(year: number) {
```

Comment:
Splitting this out as a separate function seems a bit unnecessary. At the very least this doesn't need `export`